lib/services/local/users_test.go

```diff
@@ -479,12 +479,47 @@ func TestIdentityService_UpsertWebauthnLocalAuth(t *testing.T) {
 			err := test.update(ctx, test.name, test.wal)
 			require.NoError(t, err)
 
-			want := test.wal
-			got, err := test.get(ctx, test.name)
+			wantWLA := test.wal
+			gotWLA, err := test.get(ctx, test.name)
 			require.NoError(t, err)
-			if diff := cmp.Diff(want, got); diff != "" {
+			if diff := cmp.Diff(wantWLA, gotWLA); diff != "" {
 				t.Fatalf("WebauthnLocalAuth mismatch (-want +got):\n%s", diff)
 			}
+
+			gotUser, err := identity.GetTeleportUserByWebauthnID(ctx, gotWLA.UserID)
+			require.NoError(t, err)
+			require.Equal(t, test.name, gotUser)
+		})
+	}
+}
+
+func TestIdentityService_GetTeleportUserByWebauthnID(t *testing.T) {
+	t.Parallel()
+	identity, _ := newIdentityService(t)
```

Comment:
Is the second return parameter an error? Ignore the suggestion if it isn't.
```suggestion
	identity, err := newIdentityService(t)
	require.NoError(t, err)
```

Reply:
It's a clock. It tripped me too at first, let me swap it around a bit.